src/components/HeaderWithLanguage/HeaderInformation.jsx

```diff
@@ -82,11 +82,15 @@ export const types = {
     cssModifier: 'filter',
     icon: <Filter />,
   },
+  learningpath: {
+    icon: <ContentTypeBadge type={contentTypes.LEARNING_PATH} background size={'xsmall'} />,
+  },
```

Comment:
Brukes denne til noko? Er ikkje dette for å vise type data som redigeres? Og læringssti kan ikkje redigeres i ed. (endå)

Reply:
oj nei, det var en feil. Skal fjerne den!